.docs/test_plan.md

```diff
@@ -0,0 +1,190 @@
+ # Test plan of Armonik "2.12.0"
```

Comment:
Can we not specify the version here? Because it will be roughly the same test plan for the other versions.